### DISCUSSION TITLE
Getting started from scratch?

### DISCUSSION THREAD

**Discussion by crbn60** [BradenM/micropy-cli#552] [Q&A]
Good Afternoon All,

I'm trying to bootstrap myself into a MicroPython project for ESP32. I think that means I want to be able to run the "unix" port on my workstation for development purposes, right? Then migrate to hardware for testing?

Advice is appreciated!

So far, I can't seen to install stubs:

```bash
(.venv) $ micropy init tgl-see
? Project Name tgl-see
? Choose any Templates to Generate done (5 selections)
MicroPy  You don't have any stubs!

MicroPy  To add stubs to micropy, use micropy stubs add <STUB_NAME>
MicroPy  See: micropy stubs --help for more information.
Aborted.
```

So I tried to install at least one stub:

```bash
(.venv) $ micropy stubs add micropython-esp32-esp32_generic-stubs

MicroPy  Adding micropython-esp32-esp32_generic-stubs to stubs
MicroPy  micropython_esp32_esp32_generic_stubs-1.21.0.post1: 100%|███████████████████████████████████████████████████| [120k/120k @ 417MB/s]
MicroPy  https://files.pythonhosted.org/packages/56/13/0e72f60d4bf4c9e71fbd040361bfc38a0e6556b0d67b3a834fd864f1040d/micropython_esp32_esp32_generic_stubs-1.21.0.post1.tar.gz is not a valid stub!
```
Which doesn't work (scroll right for the error).

I feel like I'm missing some basic piece of knowledge and would appreciate a push in the right direction.

A.

**Comment by Nikoh77** [comment]
Same problem here, with a rpi pico w :-(
have you resolved it some way?
